test/accelerators/NNPA/numerical/TestGRU-NNPA.cpp

```diff
@@ -0,0 +1,100 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```

Comment:
please add an updated version of this
```c++
/=============-- LSTMModel.cpp - Building LSTM Models for tests -============//
//
// Copyright 2019-2022 The IBM Research Authors.
//
// =============================================================================
//
// This file contains a function that builds an LSTM model and compiles it.
//
//===----------------------------------------------------------------------===//
``` 
for this and the other files in this dir.

Reply:
Done. Thanks!